views/login.ejs

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
```

Comment:
I have already made a single signup page to be rendered for both login and signup. I have told this before to keep only a single page. Had to spend 3 hours on some minor issue just because of two different files.
**Do not create a new file. I repeat keep one file only. The conditions are given in ejs and render methods too.**